views/js/admin_order.js

```diff
@@ -28,6 +28,37 @@ var altapay = {
         });
     },
 
+    capture_child_order: function (element) {
+        var namespace = this;
+        var dataArray = [];
+        dataArray.push({name: 'action', value: 'captureRemaining'});
+        dataArray.push({name: 'payment_id', value: $(element).data('payment-id')});
+        dataArray.push({name: 'orderid', value: $(element).data('orderid')});
+        dataArray.push({name: 'remaining_amount', value: $(element).data('remaining_amount')});
+        $.ajax({
+            type: 'POST',
+            url: $(element).data('url'),
+            data: dataArray,
```

Comment:
How is the authentication being set? Is it the cookies with a session?

Reply:
@emicha , here we are passing the URL which includes a unique security token. This token is specific to the user session which make sure that the Ajax request are legitimate
` $ajaxUrl = $this->context->link->getAdminLink('AdminPayByLink', true) . '&customer_id=' . $orderDetail->id_customer;`